test/unit/uk/gov/hmrc/exports/services/mapping/declaration/DeclarantBuilderSpec.scala

```diff
@@ -16,75 +16,57 @@
 
 package unit.uk.gov.hmrc.exports.services.mapping.declaration
 
-import org.mockito.Mockito.when
 import org.scalatest.{Matchers, WordSpec}
 import org.scalatestplus.mockito.MockitoSugar
 import testdata.ExportsDeclarationBuilder
-import uk.gov.hmrc.exports.models.Country
-import uk.gov.hmrc.exports.models.declaration.Address
-import uk.gov.hmrc.exports.services.CountriesService
 import uk.gov.hmrc.exports.services.mapping.declaration.DeclarantBuilder
 import wco.datamodel.wco.dec_dms._2.Declaration
 
 class DeclarantBuilderSpec extends WordSpec with Matchers with MockitoSugar with ExportsDeclarationBuilder {
 
-  val mockCountriesService = mock[CountriesService]
-  when(mockCountriesService.allCountries)
-    .thenReturn(List(Country("United Kingdom", "GB"), Country("Poland", "PL")))
+  private def builder = new DeclarantBuilder()
 
   "DeclarantBuilder" should {
 
     "build then add" when {
+
       "no declarant details" in {
         val model = aDeclaration(withoutDeclarantDetails())
         val declaration = new Declaration()
 
         builder.buildThenAdd(model, declaration)
 
-        declaration.getDeclarant should be(null)
+        declaration.getDeclarant shouldBe null
       }
 
-      "no eori" in {
-        val model = aDeclaration(withDeclarantDetails(eori = None, address = Some(Address("name", "line", "city", "postcode", "United Kingdom"))))
-        val declaration = new Declaration()
-
-        builder.buildThenAdd(model, declaration)
+      "empty declarant details" in {
```

Comment:
This test case and the next one are exactly the same.

Reply:
empty declarant details means empty but visited. The model is different. Behaviour the same

Reply:
Ok I see